src/uk/gov/hmcts/contino/GradleBuilder.groovy

```diff
@@ -264,7 +264,11 @@ EOF
     } else if (localSteps.fileExists("security.sh")) {
       WarningCollector.addPipelineWarning("security.sh_moved", "Please remove security.sh from root of repository, no longer needed as it has been moved to the Jenkins library", LocalDate.of(2023, 04, 17))
     } else {
-      localSteps.writeFile(file: 'security.sh', text: localSteps.libraryResource('uk/gov/hmcts/pipeline/security/backend/security.sh'))
+      if (localSteps.env.IS_FRONTEND == 'true') {
```

Comment:
where does this come from?

Reply:
Added a note into the readme. Should be passed from the Jenkinsfile

Reply:
hmm bit hacky, we normally do this via config DSL, any thoughts @adusumillipraveen ?

Reply:
yeah, recommend using config as well for the exclusion config and do decide if its frontend, can we do it based on pipeline `type`  ? if node

Reply:
Makes sense for most cases, the only issue with that is that this has been brought up by idam who use gradle for their frontend, so there's an exception there. They currently have a custom script but having some logic here would enable them to use the centralised script and just add the extra exclusion they need

Reply:
sure, it can just be optional parameters in `enablesecutiryScan`?